src/modules/textFacets/__tests__/textFacets.test.js

```diff
@@ -0,0 +1,152 @@
+import renderer from 'react-test-renderer';
+import { render, waitFor, fireEvent } from '@testing-library/react';
+import React from 'react';
+import Products from '../../products/index';
+import TextFacets from '../index';
+import UnbxdSearchWrapper from '../../../UnbxdSearchWrapper';
+import SearchBox from '../../searchBox';
+import { searchResponse } from './mocks/searchMock';
+
+const FacetItemComponent = ({ itemData, onClick }) => {
+    const { name, count, isSelected } = itemData;
+    const handleClick = () => {
+        onClick(itemData);
+    };
+
+    return (
+        <div
+            className={`UNX-facet__item ${isSelected ? '-selected' : ''}`}
+            onClick={handleClick}
+        >
+            <div className="-checkbox" />
+            <div className="-label">{name}</div>
+            <div className="-count">({count})</div>
+        </div>
+    );
+};
+
+// establish API mocking before all tests
+beforeAll(() => {
+    window.fetch = jest.fn(() => {
+        return Promise.resolve({
+            json: () => Promise.resolve(searchResponse),
+        });
+    });
+})
+
+const attributesMap = {
```

Comment:
Use wildearth's attributeMap. `min_cheapest_default_price` does not exist on wildearth feed.

Reply:
@sanjithkumar017  Does it matter when the data and response is mocked?

Reply:
@deepunbxd005  Use RRP_Price, we have ti change it everywhere,

Reply:
done